ivy/functional/frontends/tensorflow/raw_ops.py

```diff
@@ -539,6 +539,34 @@ def EuclideanNorm(*, input, axis, keep_dims=False, name="EuclideanNorm"):
 ConcatV2 = to_ivy_arrays_and_back(map_raw_ops_alias(tf_frontend.concat))
 
 
+@to_ivy_arrays_and_back
+def Conv2D(
+    *,
+    input,
+    filter,
+    strides,
+    padding,
+    data_format="NHWC",
+    dilations=[1, 1, 1, 1],
+    name="Conv2D",
+):
+    if data_format == "NDHWC":
+        strides = [1] + strides[1:-1] + [1]
+        dilations = [1] + dilations[1:-1] + [1]
+    elif data_format == "NCDHW":
+        strides = [1, 1] + strides[2:] + [1]
+        dilations = [1, 1] + dilations[2:] + [1]
+
+    return tf_frontend.nn.conv2d(
```

Comment:
https://lets-unify.ai/ivy/stateful/layers/Conv2D.html#conv2d - according to the docs we should include the argument filter_shape argument.

Reply:
Okey, I made the changes. And added filter_shape argument.